src/erlcloud_ddb2.erl

```diff
@@ -107,7 +107,11 @@
          update_item/3, update_item/4, update_item/5,
          update_global_table/2, update_global_table/3, update_global_table/4,
          update_table/2, update_table/3, update_table/4, update_table/5,
-         update_time_to_live/2, update_time_to_live/3, update_time_to_live/4
+         update_time_to_live/2, update_time_to_live/3, update_time_to_live/4,
+         list_backups/0,list_backups/1,list_backups/2, create_backup/2, create_backup/3,
```

Comment:
please put these functions in alphabetical order in the module for consistency with the others. I mean both the export placement *and* the placement in the module.